recipes/flang-activation/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "19.1.0.rc3" %}
+
+package:
+  name: flang-activation
+  version: {{ version }}
+
+build:
+  number: 0
+  # intentionally only windows (main target) & linux (debuggability)
+  skip: true  # [osx]
+
+outputs:
+  - name: flang_impl_{{ cross_target_platform }}
+    script: install_flang_impl.sh   # [unix]
+    # we don't cross-compile on windows -> no symlinks
+    requirements:
+      - flang ={{ version }}
+      - lld ={{ version }}          # [win]
+      - compiler-rt ={{ version }}
```

Comment:
Is the package without `_{{ cross_target_platform }}` needed?

Reply:
I was basing this on [`clang_linux-64`](https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/main/recipe/meta.yaml#L97), where this is specified like this. Happy to remove